README.md

```diff
@@ -25,3 +25,41 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+
+Runtime Analysis of the Divide and Conquer Sort Algorithm:
+
+The final runtime complexity of the implemented divide and conquer sorting algorithm is:
+T(n) = 3T(n/3) + (n)
+T(n) = 3(3T(n/9) + (n/3)) + (n)
+T(n) = 3^2(T)(n/9) + 4(n)
+T(n) = 9(3T(n/27) + n/9) + 4(n) 
+T(n) = 27(T)(n/27) + 13n
+....
+T(n) = 3^i (T)(n/3^i) + (3^i - 1)n/2
+
+n/3^i = 1  => i = log3(n)
+
+T(n) = 3^log3(n) T(1) + nlog3(n)
+T(n) = nT(1) + nlog3(n) 
+T(n) = n + n log3(n)
+
+O(n log3(n)) dominates O(n), but can be rewritten as:
+T(n) = n + n log3(n) ∈ O(n log n)
+
+
+
+Reasoning:
+The recurrenc is T(n) = 3T(n/3) + O(n), this is because T(n) represents
+runnng time for an input of size n. 3T(n/3) represents the recursive calls
+on the three subarrays of size n/3, and O(n) represents the work being done at each level. Specifically
+the work at each level involves adding the results of the sums from the three subarrays, which takes linear time. We are
```

Comment:
So adding 3 numbers takes linear time? Linear in the size of the original array?